ledger-service/http-json-cli/src/main/scala/com/daml/http/Config.scala

```diff
@@ -129,21 +131,22 @@ private[http] object JdbcConfig
       s"${indent}url -- JDBC connection URL,\n" +
       s"${indent}user -- database user name,\n" +
       s"${indent}password -- database user password,\n" +
-      s"${indent}createSchema -- boolean flag, if set to true, the process will re-create database schema and terminate immediately.\n" +
+      s"${indent}createSchema -- boolean flag, if set to true, the process will re-create database schema and terminate immediately. This is deprecated and replaced by schemaHandling, however if set it will always overrule schemaHandling.\n" +
+      s"${indent}schemaHandling -- option setting how the schema should be handled. Valid options are CreateSchema, Start, CreateSchemaIfNeededAndStart, CreateSchemaAndStart.\n" +
```

Comment:
Daml on SQL calls a very similar option `sql-start-mode` https://github.com/digital-asset/daml/blob/c3a3d600832bf1926a2421302e31abc1219c8f5a/ledger/sandbox-common/src/main/scala/platform/sandbox/config/PostgresStartupMode.scala#L9
Maybe consider using the same here?

Reply:
Yeah, still in favour of consistency. Even if the schema name might get lost here.

Reply:
Because we are already in the `jdbcConfig` I named it just `start-mode` (prior I even named it `startup-mode` but to preserve consistency I went for the other).